src/components/input/input.stories.mdx

```diff
@@ -33,3 +41,64 @@ You can also add `c-input` to `<select>` elements, in which case you’ll get a
     })}
   </Story>
 </Preview>
+
+## Elastic textarea
+
+Add `js-elastic-textarea` to a `<textarea>` element\* to have it expand as the user types more than will fit.
+
+If the `rows` attribute is set, it will be used as the minimum height.
```

Comment:
```suggestion
If the `data-min-rows` attribute is set, it will be used as the minimum height.
```

Reply:
Actually it does use the `rows` attribute — As part of the create function, we add the `data-min-rows` attribute, and assign it either the initial value of the `rows` attribute, or `2` if `rows` isn't set. So for the pattern library consumer, the part they care about is the `rows` attribute.

Reply:
Got it :thumbsup: